#ieeesoc feat: add severity calculation feature (tested with Shahid and Rahul)

TheSabari07 opened the pull request:
- Implemented severity level classification for all patients. 
- Tested the logic using two sample patients (Shahid and Rahul) for verification.

Devansh051 reviewed (changes requested):
Good to see That you try to implement it so fast, But what i want to see is how this severity variable can be used in hospital_final.c like based on severity the patient can be admitted try to implement in that also, and add the severity for all patients in patient.txt so it will be uniform and give any values what you feel would be correct.
Feel Free to ask Anything !! 
Thanks For Contributing !! 
Keep Contributing !!

TheSabari07 commented:
> Good to see That you try to implement it so fast, But what i want to see is how this severity variable can be used in hospital_final.c like based on severity the patient can be admitted try to implement in that also, and add the severity for all patients in patient.txt so it will be uniform and give any values what you feel would be correct. Feel Free to ask Anything !! Thanks For Contributing !! Keep Contributing !!

Thanks for the quick review, @Devansh051!

I'll extend the logic to `hospital_final.c` so that severity directly influences hospital assignment (e.g., prioritize Red patients).

Also, I’ll update all patient records in `patient.txt` with appropriate severity values to keep things consistent.

Will push the updated commit shortly. Appreciate the feedback!

TheSabari07 commented:
> Good to see That you try to implement it so fast, But what i want to see is how this severity variable can be used in hospital_final.c like based on severity the patient can be admitted try to implement in that also, and add the severity for all patients in patient.txt so it will be uniform and give any values what you feel would be correct. Feel Free to ask Anything !! Thanks For Contributing !! Keep Contributing !!

@Devansh051 

I've updated my pull request:
-  Added severity-based hospital recommendation logic in patients_main.c.
-  The system now dynamically suggests hospitals based on severity and casualty levels.

Please review and let me know if any changes are needed. Happy to make improvements!

Devansh051 reviewed (changes requested):
Sorry but i did'nt understand what you are trying to do in the hospital_final.c file like you are taking the input as severity   for the patient from the user if it is not present , and again you are asking the severity in the case of emergency. And second thing why you are not adding the cost for the severity in non-emergency like for emergency if you are doing then why not in non-emergency. 
The other thing is that i am saying to add severity field to all the user / patients present in the patient_details.txt. So by which all patients will be having severity field.
Please tell me the reason !! 
Thanks For Contributing !! 
Feel Free to Contact !!

Devansh051 commented:
Hello Bro i am quite a bit confused regarding this can you give me your discord name so i will contact to you, most probably in the evening

TheSabari07 commented:
> Hello Bro i am quite a bit confused regarding this can you give me your discord name so i will contact to you, most probably in the evening

sure bro 
mahi7781rsd my discord username

Devansh051 commented:
please come live on discord tomorrow

Devansh051 commented:
Sorry but your pr is not able to calculate the optimal cost it is showing 0 and try to update it also and please check it completely once before doing Pr. 
Thanks  For Contributing !!

TheSabari07 commented:
> please come live on discord tomorrow

@Devansh051  sure bro 
your id ?
sorry for late reply

Devansh051 commented:
Can you call me tomorrow whenever you are free to talk like from 10:00 a.m. to 9 pm between that you can choose anytime  what I feel is if you call at 12:00 a.m. so it would be better because I am mostly free at that time so you can call at the time or you can do one thing if I didn't pick the call put leave the message in that this chat of discord   of your contact number I will contact to you from my side only if you are ok which sharing in the contact number
